### DISCUSSION TITLE
Playwright version

### DISCUSSION THREAD

**Discussion by imartinflores** [XRM-OSS/D365-UI-Test#42] [General]
hey guys, @DigitalFlow , @julianYaman - I think Playwright version should be updated. It is on 1.19 (almost 20 to be released) - there are a lot of improvements specially on "locator" that could allow us to select elements on the grid based on description or any text for example instead of an index. Have you checked ?

https://www.youtube.com/watch?v=z0EOFvlf14U

**Comment by DigitalFlow** [comment]
Hi @imartinflores,

thanks for the input :)
Sounds promising. From my side, let's upgrade as soon as 1.20 is out :+1:
